release/src/main/scripts/build_release_candidate.sh

```diff
@@ -49,21 +47,28 @@ PYTHON_VER=("python2.7" "python3.5" "python3.6" "python3.7")
 FLINK_VER=("1.7" "1.8" "1.9")
 
 echo "================Setting Up Environment Variables==========="
-echo "Which release version are you working on: "
-read RELEASE
+if [[ "$RELEASE" == "" ]]; then
+  echo "Which release version are you working on (e.g. 2.18.0): "
```

Comment:
TBH I'd be happy bailing out here. OK to leave it, too, but let the user know how they can avoid the interactivity (I will always just Ctrl-C and set the env var)